.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea/*
 *.iml
 target/*
+src/main/java/core/basesyntax/util/*
+src/main/java/*.java
```

Comment:
do we need this?

Reply:
if I'd like to leave this files locally, I guess yes

Reply:
done